src/sentry/relay/config/__init__.py

```diff
@@ -70,7 +74,9 @@ def get_exposed_features(project: Project) -> Sequence[str]:
     return active_features
 
 
-def get_public_key_configs(project, full_config, project_keys=None):
+def get_public_key_configs(
+    project: Project, full_config: bool, project_keys: Optional[Sequence[ProjectKey]] = None
+) -> Sequence[Mapping[str, Any]]:
```

Comment:
Any reason we use `List` for other return types but `Sequence` here? I think we can make them all `List`, because we know there's an actual list returned.

Reply:
general advice is: take parameters as abstract types, return values as concrete types -- this allows the most flexibility for apis while also being concrete about the return values